meet_balls_app/tests.py

```diff
@@ -22,3 +23,47 @@ def test_display_game_event_form(self, client):
         response = client.get('/game-events/create/')
         assert response.status_code == 200
         assert b'<form' in response.content
+
+
+@pytest.mark.django_db
+class Testlogin:
+    def test_successful_login(self, client, player):
+        username = "daniel"
+        password = "password"
+
+        login_url = reverse('loginUser')
+        response = client.post(
+            login_url, {'username': username, 'password': password})
+
+        assert response.templates[0].name == 'meet_balls_app/home.html'
+        assert response.status_code == 200
+        assert response.context['user']['is_authenticated'] is True
+        assert response.context['user']['username'] == username
+
+    @pytest.mark.parametrize('username, password', [
+        ('wronguser', 'wrongpass'),  # Incorrect username and password
+        ('daniel', 'wrongpass'),  # Correct username, incorrect password
+        ('wronguser', 'password'),  # Incorrect username, correct password
```

Comment:
You picked self-explanatory usernames and passwords. You don't need the comments.

Reply:
@yftacherzog removed